Extensions/ValuesOfMultipleObjects.json

```
@@ -0,0 +1,1796 @@
+          "supplementaryInformation": "",
+          "type": "objectList"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "X center point of picked object instances (middle)",
+      "fullName": "X center point of picked object instances (middle)",
+      "functionType": "Expression",
+      "name": "CenterOfObjectsX_Middle",
+      "private": false,
+      "sentence": "",
+      "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [],
+          "actions": [
+            {
+              "type": {
+                "inverted": false,
+                "value": "SetReturnNumber"
+              },
+              "parameters": [
+                "(ValuesOfMultipleObjects::MaxX_Center(Object) + ValuesOfMultipleObjects::MinX_Center(Object)) / 2"
+              ],
+              "subInstructions": []
+            }
+          ],
+          "events": []
+        }
+      ],
+      "parameters": [
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Objects",
+          "longDescription": "Objects that will be used to calculate their center point ",
+          "name": "Object",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "objectList"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Y center point of picked object instances (middle)",
+      "fullName": "Y center point of picked object instances (middle)",
+      "functionType": "Expression",
+      "name": "CenterOfObjectsY_Middle",
+      "private": false,
+      "sentence": "",
+      "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [],
+          "actions": [
+            {
+              "type": {
+                "inverted": false,
+                "value": "SetReturnNumber"
+              },
+              "parameters": [
+                "(ValuesOfMultipleObjects::MaxY_Center(Object) + ValuesOfMultipleObjects::MinY_Center(Object)) / 2"
+              ],
+              "subInstructions": []
+            }
+          ],
+          "events": []
+        }
+      ],
+      "parameters": [
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Objects",
+          "longDescription": "Objects that will be used to calculate their center point ",
+          "name": "Object",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "objectList"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "X center point of picked object instances (average)",
+      "fullName": "X center point of picked object instances (average)",
+      "functionType": "Expression",
+      "name": "CenterOfObjectsX_Average",
```

```suggestion
      "name": "AverageCenterOfObjectsX",
```

I think that the underscoring sounds not very user-friendly, a more english-sentence-like name offers imo a better experience and is easier to remember.

---

I agree, I proposed some non-underscore versions in the previous suggestion.